### PR TITLE
RHDEVDOCS-2493 Logging: Change name from "cluster logging" to "OpenSh… [enterprise-4.7]

### PR DESCRIPTION
**Pull request by rolfedh**
…ift Logging"
https://issues.redhat.com/browse/RHDEVDOCS-2493

Replaced most instances of "cluster logging" with "OpenShift Logging," as requested, with the following exceptions:

- "Cluster Logging Custom Resource"
- "cluster logging operator"
- References of "Cluster Logging" in the product UI, such as in "Choose  *Cluster Logging* from the list of available Operators, and click *Install*."

### CODE REVIEW

File: _unused_topics/cluster-logging-exported-fields-docker.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-exported-fields-container_{context}"]
 = Container exported fields
 
-These are the Docker fields exported by the {product-title} cluster logging available for searching from Elasticsearch and Kibana.
+These are the Docker fields exported by the OpenShift Logging available for searching from Elasticsearch and Kibana.
```
Comment (ewolinetz) [inline _unused_topics/cluster-logging-exported-fields-docker.adoc:8]: what values can `{product-title}` be again?

File: logging/cluster-logging-deploying.adoc [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
 toc::[]
 
 
-You can install cluster logging by deploying
-the Elasticsearch and Cluster Logging Operators. The Elasticsearch Operator
-creates and manages the Elasticsearch cluster used by cluster logging. 
-The Cluster Logging Operator creates and manages the components of the logging stack. 
+You can install OpenShift Logging by deploying
+the Elasticsearch and cluster logging operators. The Elasticsearch Operator
```
Comment (ewolinetz) [inline logging/cluster-logging-deploying.adoc:10]: "Cluster Logging"

File: logging/cluster-logging-deploying.adoc [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+creates and manages the Elasticsearch cluster used by OpenShift Logging. 
+The cluster logging operator creates and manages the components of the logging stack.
```
Comment (ewolinetz) [inline logging/cluster-logging-deploying.adoc:12]: "Cluster Logging Operator"
https://github.com/openshift/cluster-logging-operator/

File: _unused_topics/cluster-logging-exported-fields-docker.adoc (hunk 1 above)
Comment (rolfedh) [reply _unused_topics/cluster-logging-exported-fields-docker.adoc:8]: @ewolinetz `{product-title}` is defined as:
`:product-title: OpenShift Container Platform`

File: logging/cluster-logging-deploying.adoc (hunk 3 above)
Comment (rolfedh) [reply logging/cluster-logging-deploying.adoc]: > "Cluster Logging Operator"
> https://github.com/openshift/cluster-logging-operator/

Thanks. I've capitalized "Cluster Logging Operator" throughout the docs.

File: logging/cluster-logging-deploying.adoc (hunk 2 above)
Comment (rolfedh) [reply logging/cluster-logging-deploying.adoc]: @ewolinetz Sorry - I don't understand which string I should change to "Cluster Logging." Would you clarify?

**Comment by rolfedh** [comment]
@alexissolanas @linuxstudio @ptrknvk Please take a look. Thanks.

File: _unused_topics/cluster-logging-exported-fields-docker.adoc (hunk 1 above)
Comment (bobfuru) [inline _unused_topics/cluster-logging-exported-fields-docker.adoc:8]: In this example, "the OpenShift Logging" sounds odd to me. Suggest dropping "the"

File: _unused_topics/cluster-logging-uninstall-cluster-ops.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-uninstall-ops_{context}"]
 = Uninstall the infra cluster
 
-You can uninstall the infra cluster from the {product-title} cluster logging. 
+You can uninstall the infra cluster from the OpenShift Logging.
```
Comment (bobfuru) [inline _unused_topics/cluster-logging-uninstall-cluster-ops.adoc:8]: In this example, "the OpenShift Logging" sounds odd to me. Suggest dropping "the"

File: logging/config/cluster-logging-configuring-cr.adoc [hunk 5]
```diff
@@ -1,11 +1,11 @@
 :context: cluster-logging-configuring-cr
 [id="cluster-logging-configuring-cr"]
-= About the Cluster Logging custom resource
+= About the Cluster Logging Custom Resource
```
Comment (bobfuru) [inline logging/config/cluster-logging-configuring-cr.adoc:3]: We use lowercase for "custom resource" so I think this was correct before. Seems it's  not called out explicitly in our guidelines, though I thought it was: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references
(Same with "custom resource definition (CRD)", FYI)

File: modules/cluster-logging-curator-schedule.adoc [hunk 6]
```diff
@@ -5,12 +5,12 @@
 [id="cluster-logging-curator-schedule_{context}"]
 = Configuring the Curator schedule
 
-You can specify the schedule for Curator using the `Cluster Logging` custom resource
-created by the cluster logging installation.
+You can specify the schedule for Curator using the `OpenShift Logging` custom resource
+created by the OpenShift Logging installation.
```
Comment (bobfuru) [inline modules/cluster-logging-curator-schedule.adoc:9]: IIUC, shouldn't it still be "`Cluster Logging` custom resource"?

File: modules/cluster-logging-elasticsearch-retention.adoc [hunk 7]
```diff
@@ -8,7 +8,7 @@
 You can specify how long the default Elasticsearch log store keeps indices 
 using a separate _retention policy_ for each of the three log sources: 
 infrastructure logs, application logs, and audit logs. The retention policy, 
-which you configure using the `maxAge` parameter in the Cluster Logging Custom 
+which you configure using the `maxAge` parameter in the OpenShift Logging Custom 
 Resource (CR), is considered for the Elasticsearch roll over schedule and
```
Comment (bobfuru) [inline modules/cluster-logging-elasticsearch-retention.adoc:12]: s/Custom Resource/custom resource

File: modules/cluster-logging-log-forwarding-disable.adoc [hunk 8]
```diff
@@ -5,18 +5,18 @@
 [id="cluster-logging-log-forwarding-disable_{context}"]
 = Disabling the Log Forwarding feature
 
-To disable the Log Forwarding feature, remove the `clusterlogging.openshift.io/logforwardingtechpreview:enabled` parameter from the Cluster Logging custom resource (CR) and delete the `ClusterLogForwarder` CR. The container and node logs will be forwarded to the internal {product-title} Elasticsearch instance.
+To disable the Log Forwarding feature, remove the `clusterlogging.openshift.io/logforwardingtechpreview:enabled` parameter from the Cluster Logging Custom Resource (CR) and delete the `ClusterLogForwarder` CR. The container and node logs will be forwarded to the internal {product-title} Elasticsearch instance.
```
Comment (bobfuru) [inline modules/cluster-logging-log-forwarding-disable.adoc:8]: s/Custom Resource/custom resource

File: modules/jaeger-install-elasticsearch.adoc [hunk 9]
```diff
@@ -20,7 +20,7 @@ Do not install Community versions of the Operators. Community Operators are not
 
 [NOTE]
 ====
-If you have already installed the Elasticsearch Operator as part of OpenShift cluster logging, you do not need to install the Elasticsearch Operator again.  The Jaeger Operator will create the Elasticsearch instance using the installed Elasticsearch Operator.
+If you have already installed the Elasticsearch Operator as part of OpenShift OpenShift Logging, you do not need to install the Elasticsearch Operator again.  The Jaeger Operator will create the Elasticsearch instance using the installed Elasticsearch Operator.
```
Comment (bobfuru) [inline modules/jaeger-install-elasticsearch.adoc:23]: "OpenShift" appears twice here

File: modules/serverless-using-cluster-logging-find-logs-services-deployed.adoc [hunk 10]
```diff
@@ -2,9 +2,9 @@
 //
 // serverless/cluster-logging-serverless.adoc
 [id="using-cluster-logging-to-find-logs-for-services-deployed-with-knative-serving_{context}"]
-= Using cluster logging to find logs for services deployed with Knative Serving
+= Using OpenShift Logging to find logs for services deployed with Knative Serving
 
-With OpenShift Cluster Logging, the logs that your applications write to the console are collected in Elasticsearch. The following procedure outlines how to apply these capabilities to applications deployed by using Knative Serving.
+With OpenShift OpenShift Logging, the logs that your applications write to the console are collected in Elasticsearch. The following procedure outlines how to apply these capabilities to applications deployed by using Knative Serving.
```
Comment (bobfuru) [inline modules/serverless-using-cluster-logging-find-logs-services-deployed.adoc:7]: OpenShiftx2

File: modules/gathering-data-specific-features.adoc [hunk 11]
```diff
@@ -42,7 +42,7 @@ ifndef::openshift-origin[]
 |Data collection for Red Hat OpenShift Container Storage.
 
 |`registry.redhat.io/openshift4/ose-cluster-logging-operator`
-|Data collection for Red Hat OpenShift cluster logging.
+|Data collection for Red Hat OpenShift OpenShift Logging.
```
Comment (bobfuru) [inline modules/gathering-data-specific-features.adoc:45]: OpenShift x2

**Review by bobfuru — COMMENTED** [review]
Some small items to fix but overall looks good.

File: modules/dedicated-cluster-install-deploy.adoc [hunk 12]
```diff
@@ -124,13 +124,13 @@ spec:
 .. Click *Create* to deploy the logging instance, which creates the Cluster
 Logging and Elasticsearch Custom Resources.
```
Comment (bobfuru) [inline modules/dedicated-cluster-install-deploy.adoc:125]: Should this be "custom resources"?

**Comment by bobfuru** [comment]
LGTM

**Pull request merged**